cmd/main.go

```diff
@@ -14,16 +14,16 @@ import (
 var log = logging.WithFields(logging.Fields{})
 
 var opts struct {
-	Port          int    `long:"port" short:"p" default:"9090" description:"Port on which to serve."`
-	Region        string `long:"region" short:"r" env:"AWS_REGION" required:"true" description:"AWS region name"`
-	Profile       string `long:"profile" short:"f" env:"AWS_PROFILE" default:"" description:"Named AWS profile to be used"`
-	RefreshPeriod int    `long:"refresh-period" default:"360" description:"Refresh period in seconds"`
+	Port           int      `long:"port" short:"p" default:"9090" description:"Port on which to serve."`
+	Region         string   `long:"region" short:"r" env:"AWS_REGION" required:"true" description:"AWS region name"`
+	Profile        string   `long:"profile" short:"f" env:"AWS_PROFILE" default:"" description:"Named AWS profile to be used"`
+	RefreshPeriod  int      `long:"refresh-period" default:"360" description:"Refresh period in seconds"`
+	IncludeAWSTags []string `long:"include-aws-tags" description:"The aws resource tags to include as labels for returned metrics"`
```

Comment:
super nit: Should that be singular, since we add one tag at a time? (eg. --include-aws-tag tag1 --include-aws-tag tag2)